spec/spec_helper.rb

```diff
@@ -0,0 +1,15 @@
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..'))
+
+require 'dotenv'
+Dotenv.load
```

Comment:
I am loading my environment variables twice in the application. Once here, in the spec_helper.rb, and once in config.ru. If I didn't load them in spec_helper.rb my API calls in my tests would fail. Do you know of a solution where I would only need to load them once?